Description: Remove decorators from slots
 .
 They apparently make the entire UI just crash since the last update of
 PyQt. Removing them.
 .
 weborf (0.13.3-2) unstable; urgency=medium
 .
   * Remove decorators to workaround a bug in PyQt
Author: Salvo 'LtWorf' Tomaselli <tiposchi@tiscali.it>
--- weborf-0.13.3.orig/qweborf/whelper.py
+++ weborf-0.13.3/qweborf/whelper.py
@@ -140,7 +140,6 @@ class weborf_runner():
         self.listener.new_socket.connect(self.socket_cback)
         self.listener.start()
 
-    @QtCore.pyqtSlot(object)
     def socket_cback(self, sock):
         '''Recieves connection requests and decides if they have to be authorized or denied'''
 
@@ -235,7 +234,6 @@ class weborf_runner():
             logentry = 'Address: <a href="%s">%s</a>' % (url, url)
             self.logclass.logger(logentry)
 
-    @QtCore.pyqtSlot(object, int)
     def _child_terminated(self, child, exit_code):
         '''Called when the child process is terminated
         param child is for now ignored'''
